mojo/core/channel_posix.cc

```diff
@@ -3,6 +3,9 @@
 // found in the LICENSE file.
 
 #include "mojo/core/channel.h"
+#define DISABLE_MULTI_CONNECTION_CHANGES 0
+// TODO temporarily disable for resolving resoure loading issue with
+// networkservice
 
```

Comment:
Instead of defining DISABLE_MULTI_CONECTION_CHANGES on all files. Can we move to 
"base/memory/shared_memory_tracker.h" ?

Then no need to define again and again, as all these files includes "base/memory/shared_memory_tracker.h".